driver-core/src/test/unit/com/mongodb/operation/AsyncQueryBatchCursorSpecification.groovy

```diff
@@ -429,7 +429,7 @@ class AsyncQueryBatchCursorSpecification extends Specification {
         new ServerVersion([3, 0, 0]) | false                | queryResult(SECOND_BATCH)
     }
 
-    def 'should kill the cursor in the getMore callback if it was closed before getMore returned'() {
+    def 'should not kill the cursor in the getMore callback if it was closed before getMore returned'() {
```

Comment:
Generally, this doesn't seem to test all the changes that were made.  Did you/we determine that it wasn't feasible to do so?

Reply:
I ran the test with the debugger, and the only changes that were not covered was when an exception was thrown in `getMore` and `close` was called. I have added another test to simulate that situation.